torch/optim/adadelta.py

```diff
@@ -198,7 +226,9 @@ def adadelta(
     """
     # We still respect when the user inputs False for foreach.
     if foreach is None:
-        _, foreach = _default_to_fused_or_foreach(params, differentiable, use_fused=False)
+        _, foreach = _default_to_fused_or_foreach(
```

Comment:
What linter are you using and why is it relinting already linted lines?

Reply:
lintrunner on my devgpu? haha, not sure

Reply:
nah, it looks like your max line width is too narrow